aten/src/ATen/native/TensorShape.cpp

```diff
@@ -1349,6 +1371,23 @@ std::vector<Tensor> unsafe_split_with_sizes(const Tensor& self, IntArrayRef spli
   return result;
 }
 
+std::vector<Tensor> hsplit(const Tensor& self, IntArrayRef split_sizes, c10::optional<int64_t> dummy) {
+  TORCH_CHECK(!dummy.has_value(), "torch.hsplit expects dummy variable never take a value.")
```

Comment:
try to map the previous discussion to these error messages, too; let me know if you have any questions